ds-prep-python-functions/main.py

```diff
@@ -0,0 +1,42 @@
+from pyrsistent import b
+
+bool_to_int = lambda value: int(value)
+print(bool_to_int(True))
+print(bool_to_int(False))
+
+get_smaller = lambda a,b: min (a,b)
+print ("get smaller:", (get_smaller(16,31)))
+print ("get smaller:", (get_smaller(253,223)))
+
+def cube (base):
+  return (base**3)
+print  (cube(3))
+print  (cube(5))
+
+def absolute_difference (a, b) :
+  return abs ( a,b )
+print ("absolute difference:", (abs (14-11)))
+print ("absolue difference:", (abs (13-40)))
+
+def squared_difference (a, b):
+  return ((abs (a,b))**2)
+print ("square_difference:", ((abs(14,11))**2))
```

Comment:
Like above, please make sure you're calling the `squared_difference` function to get the value, not calculating it manually outside the function definition.